lib/src/syn/v2/parser/stmt/define.rs

```diff
@@ -357,6 +358,7 @@ impl Parser<'_> {
 		let mut res = DefineFieldStatement {
 			name,
 			what,
+			permissions: Permissions::none(),
```

Comment:
These permissions can stay full (default). All resources are submissive by default, with the exception of tables which should be restrictive by default